application/components/search/SearchEngine.php

```diff
@@ -36,7 +36,7 @@ public function doSearch($query)
         $resultset = $this->doSearchInternal();
         if ($resultset === null) {
             $this->doNotifyOutage();
-        } else {
+        } else if (YII_ENV != 'dev') {
```

Comment:
Can you please add a `defined("YII_ENV")` check here before comparing its value? Make sure that if the stage variable is prod it still works.

Reply:
I have done it like this:
`        
else if (defined("YII_ENV") and YII_ENV != 'dev') {
    $this->logQuery($resultset->getCount());
}
`
If I convert the stage variable in .env.local config file to `prod`, it gives error:
`An internal server error occurred.`